File: Modul 4/Soal 5/PRAK305-2210817210006-FarlyhaydyH.Djalil.c
#include <stdio.h>

int main()
{
    int i=0;
    while(i<=4){
    int h,j,d,i,m;
    scanf("%d", &i);
    h=i/86400;
    j=(i/3600)%24;
    m=(i/60)%60;
    d=i%60;
    if(i<=3600){
    printf("%.2d:%.2d:%.2d\n\n", j,m,d);
    }else if(i>3600 && i<86400){
    printf("%.2d:%.2d:%.2d\n\n", j,m,d);
    }else if(i>=86400){
    printf("%d hari %.2d:%.2d:%.2d\n", h,j,m,d);
    }
    i++;
    }
}

File: Modul 4/Soal 5/PRAK305-2210817210006-Farlyhaydy H.Djalil.py
I=0
while True:
    I=I+1
    i =int(input())
    h =int(i/84600)
    j =int((i/3600) % 24)
    m =int((i/60) % 60) 
    d =int(i % 60)
    if (i <= 3600):
        print("%.2d:%.2d:%.2d\n\n" % (j,m,d))
    elif (i >= 3600 and i < 84600):
        print("%.2d:%.2d:%.2d\n\n" % (j,m,d))
    elif (i >= 84600):
        print("%d hari %.2d:%.2d:%.2d\n" % (h,j,m,d))
    if (I>4) :
        break
    